charts/argo-cd/README.md.gotmpl

```diff
@@ -39,6 +39,9 @@ repoServer:
   autoscaling:
     enabled: true
     minReplicas: 2
+
+applicationSet:
+  replicaCount: 2
```

Comment:
Shouldn't this be also changed to `replicas` as everywhere else?

Reply:
I think both new components of 2.3 (notification & appset controller) does not yet support HA?